var/spack/repos/builtin/packages/py-fitter/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class PyFitter(PythonPackage):
+    """fitter package provides a simple class to identify the distribution
+    from which a data samples is generated from. It uses 80 distributions
+    from Scipy and allows you to plot the results to check what is the
+    most probable distribution and the best parameters."""
+
+    homepage = "https://github.com/cokelaer/fitter"
+    pypi = "fitter/fitter-1.5.1.tar.gz"
+
+    # notify when the package is updated.
+    # maintainers = ["carsonwoods"]
```

Comment:
Did you intend to remove the comment from this line so Spack can process the `maintainers` list?

Reply:
Oh yes, that was an oversight. I've gone ahead and changed it.